src/main/java/com/amazon/opendistroforelasticsearch/ad/AnomalyDetectorPlugin.java

```diff
@@ -201,7 +201,9 @@
 
     private static final Logger LOG = LogManager.getLogger(AnomalyDetectorPlugin.class);
 
-    public static final String AD_BASE_URI = "/_opendistro/_anomaly_detection";
+    public static final String AD_BASE_LEGACY = "/_opendistro/_anomaly_detection";
```

Comment:
I was wondering whether these should be `AD_BASE_LEGACY` or `LEGACY_AD_BASE` and I decided in https://github.com/opensearch-project/job-scheduler/pull/19/files to make it `LEGACY_AD_BASE` because I wanted those to be grouped together. Furthermore I think these should include OPENDISTRO in them. So to be consistent I propose that `AD_BASE_URI` becomes `LEGACY_OPENDISTRO_AD_BASE_URI`. 

Whether we do  `LEGACY_OPENDISTRO_AD_BASE_URI` or `AD_BASE_URI_LEGACY_OPENDISTRO` is not important to me, but let's decide the convention now?

Reply:
I actually like prefixing `LEGACY` it makes more sense rather than postfixing it :). 
Sure definitely, lets do `LEGACY_OPENDISTRO_AD_BASE_URI` it makes more sense to me.